distributions/otelcol-contrib/manifest.yaml

```diff
@@ -101,6 +101,7 @@ processors:
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/servicegraphprocessor v0.88.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/spanmetricsprocessor v0.88.0
   - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/spanprocessor v0.88.0
+  - gomod: github.com/open-telemetry/opentelemetry-collector-contrib/processor/sumologicprocessor v0.89.0
```

Comment:
I think set it to the same as others `v0.88.0`, it will be bumped to v0.89.0 in the release process.

Other examples:
- https://github.com/open-telemetry/opentelemetry-collector-releases/pull/389/files
- https://github.com/open-telemetry/opentelemetry-collector-releases/pull/405/files

Reply:
It won;t build anyway as there is no such version of processor yet :grimacing:

Reply:
for example [commit with routing connector ](https://github.com/open-telemetry/opentelemetry-collector-releases/commit/3d5a87e30a250dbf0df686a8d3dc3efe3de80f4a) was prepared on 19th September but [v0.85](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.85.0) was released on 12th September so there was release with routing connector, probably this kind of pull request should be prepared after release of otelcol-contirb with component enabled 🤔

Reply:
All right then, need to rebase this when v0.89.0 is released 👍